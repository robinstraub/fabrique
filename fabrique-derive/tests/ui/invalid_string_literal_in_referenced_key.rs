use fabrique_derive::Factory;

#[derive(Factory)]
struct Anvil {
    #[fabrique(relation = "Hammer", referenced_key = "")]
    hammer_id: u32,
    weight: u32,
}

fn main() {}