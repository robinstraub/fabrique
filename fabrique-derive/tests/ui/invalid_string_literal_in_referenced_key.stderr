error: Unknown value: `` at referenced_key
 --> tests/ui/invalid_string_literal_in_referenced_key.rs:4:1
  |
4 | / struct Anvil {
5 | |     #[fabrique(relation = "Hammer", referenced_key = "")]
6 | |     hammer_id: u32,
7 | |     weight: u32,
8 | | }
  | |_^
